module-types/Datacom/PSU-400-DC-F.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 400 DC-F
+part_number: 800.5215.xx
+comments: DC power supply with 48V-60Vdc input. Forced ventilation with airflow exiting through the front panel of the power supply. Supports hot-swap and
+  operation in redundant mode with another compatible PSU.
+power-ports:
+  - name: PSU
```

Comment:
Please update this name to use the `{module}` definition

Reply:
sorry, but I don't understand how it should be